docs/guides/admin/docs/configuration/https/apache-httpd.md

```diff
@@ -0,0 +1,79 @@
+Enable HTTPS using Apache httpd
+===============================
+
+> [Using Nginx as reverse proxy for Opencast](nginx.md) is the preferred way of running Opencast.
+> Refer to the [Nginx guide for configuration instructions](nginx.md) for that type of setup.
+
+This guide will help you to configure httpd to act as HTTP(S) proxy for Opencast.
+
+
+Opencast Configuration
+----------------------
+
+Make sure to use `https` as protocol for `org.opencastproject.server.url` in `etc/custom.properties`.
+
+```ini
+org.opencastproject.server.url=https://example.opencast.org
+```
+
+Also enable secure cookies in `etc/org.ops4j.pax.web.cfg`:
+
+```ini
+org.ops4j.pax.web.session.cookie.secure=true
+org.ops4j.pax.web.session.cookie.httpOnly=true
+```
+
+No other configuration is required.
+Do not enable TLS in Opencast.
+Listen to local connections only.
+Both are the default settings.
+
+
+Minimal Set-up
+--------------
+
+> Note that this guide does not give any security advice but is meant to provide a minimal working example which works
+> well with Opencast.
+
+The following configuration is an example for `/etc/httpd/conf.d/opencast.conf`.
+Note that depending on your distributions packaging, often `conf.d` or `sites-enabled` directories are used.
+Adjust the file path accordingly.
+
+Explanations for the configuration directives are provided inline. Please make sure to replace `example.opencast.org`
+with your node's domain name.
+
+The main goals of this set-up are:
+
+- Always redirect to HTTPS
+- Proxy to Opencast and take care of TLS
+- Avoid caching
+
+
+```
+<VirtualHost *:80>
+  ServerName opencast.lkiesow.io
```

Comment:
This should be example.opencast.org as mentioned in line 42 to be consistent

Reply:
Fixed